### PR TITLE
[R4R]BEP-1: update the BEP workflow.

### PR DESCRIPTION
**Pull request by brilliant-lx**
## Description
The current BEP guideline has not been updated fort a long time, it is a living document and some improvements need to be made.

## Specification
Mainly 2 updates to the BEP1:
**1.The three BEP Types are renamed.**
Feature Request -> Standards,
Improvement -> Information, 
Standard -> Process

**2.A new BEP workflow is introduced.**
image

### CODE REVIEW

File: BEP1.md
Comment (RenRick) [inline BEP1.md:19]: "on an issue" -> "on issues", "Because the BEP"-> "Because BEPs"

Comment (RenRick) [reply BEP1.md:19]: "their revisions history is the historical record of the feature proposal" -> "their revisions are the historical records of feature proposals"

File: BEP1.md
Comment (RenRick) [inline BEP1.md:21]: “BEPs are a convenient way to track the progress of their implementation.” ->" it is a convenient way to track the progress of their implementation by BEPs"

Comment (RenRick) [reply BEP1.md:21]: "This will give end users a convenient way to know the current status of a given feature, function or improvement."->"It will help end users to know the status of a given feature, function or improvement"

File: BEP1.md
Comment (guagualvcha) [inline BEP1.md:40]: In which phase, the BEP can be merged?

Comment (brilliant-lx) [reply BEP1.md:40]: As long as long BEP is broadcasted to the BEP community and no against voice is received. It can be merged as long as 2 reviewer is approved.

File: BEP1.md
Comment (brilliant-lx) [reply BEP1.md]: accepted and updated

File: BEP1.md
Comment (brilliant-lx) [reply BEP1.md]: accepted and updated

File: BEP1.md
Comment (brilliant-lx) [reply BEP1.md]: updated

**Review by RenRick — APPROVED** [review]
LGTM

File: BEP1.md
Comment (yutianwu) [inline BEP1.md:39]: why do we need a `pre-draft` status, when the pr is raised, we can view it as a draft and we can  review and discuss about the proposed BEP

Comment (brilliant-lx) [reply BEP1.md:39]: pre-draft is not a BEP, it needs to be accepted and it will enter draft state with a BEP number

File: BEP1.md
Comment (brilliant-lx) [reply BEP1.md]: IMO, to be a member of the committee, the person should be know the bnb chain very well, deliver some contribution and be influential first.
As the first steps,  some of the core developers will be selected to form the committee 1.0.
Validators and other participants are welcomed to be the committee member in the future, as long as they demonstrate our influence.

Comment (brilliant-lx) [reply BEP1.md]: There may not have such kind of organization as committee, it will make things more complicate and centralized, i.e. a small group of people have the right to determine the future of the BNB Chain, it is not what I want.
What we need to do is make BEP more transparent and fully discussed, not enable it in a hurry(except emergent cases).

**Review by yutianwu — DISMISSED** [review]

**Review by yutianwu — APPROVED** [review]

**Review by RenRick — APPROVED** [review]
LGTM

**Review by KeefeL — APPROVED** [review]
LGTM

**Pull request merged**
